### PR TITLE
Add Dokuwiki integration

### PR DESCRIPTION
**Pull request by mskrip**
Creates toggl button anywhere on the site. 
Button is placed right bellow the header node and can be used for any activity on dokuwiki (reading, editing, comparing...)

screenshot from 2017-07-10 17-41-56
screenshot from 2017-07-10 17-42-03

### CODE REVIEW

File: src/scripts/content/dokuwiki.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    pName = numElem.textContent.split(":")[0].trim();
+
+  description = numElem.textContent.split(" ").pop().trim();
+  link = togglbutton.createTimerLink({
+    className: 'wiki',
+    description: description,
+    projectName: pName
+  });
+
+  $(".wrapper.group, .pageId").prepend(link);
```
Comment (Lubo) [inline src/scripts/content/dokuwiki.js]: I'd do something like:

```javascript
var target = $(".wrapper.group") // DokuWiki Template
           || $(".pageId");      // Bootstrap3 Template

target.prepend(link);
```

To  pin-point which selector is effective of which template.

File: src/scripts/origins.json [hunk 2]
```diff
@@ -101,6 +101,11 @@
     "name": "Doit",
     "clone": "true"
   },
+  "dokuwiki.org": {
+    "url": "*://*.dokuwiki.org/*",
+    "name": "Dokuwiki",
```
Comment (Lubo) [inline src/scripts/origins.json]: It's `DokuWiki`. Just sayin' :)

**Comment by mskrip** [comment]
This is a screenshot for bootstrap version
screenshot from 2017-07-10 19-59-42

File: src/scripts/content/dokuwiki.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/*jslint indent: 2, unparam: true*/
+/*global $: false, document: false, togglbutton: false*/
+
+'use strict';
+togglbutton.render('#dokuwiki__content', {observe: false}, function (elem) {
+  var link, description,
+    numElem = $(".pageId span"),
+    pName = numElem.textContent.split(":")[0].trim(),
+    target = $(".wrapper.group") || // DocuWiki Template
```
Comment (Lubo) [inline src/scripts/content/dokuwiki.js]: It's `DokuWiki Template`.

File: src/scripts/content/dokuwiki.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    $(".pageId");                   // Bootstrap3 Template
```
Comment (Lubo) [inline src/scripts/content/dokuwiki.js]: Indentation does not seem right. Please fix it.

Comment (mskrip) [inline src/scripts/content/dokuwiki.js]: That's how the system's linter wants it indented.

**Comment by mskrip** [comment]
Hi @syko would it be possible to have this #849 and #851 integrations reviewed? Thank you

**Pull request merged**